client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx

```diff
@@ -38,26 +40,64 @@ class InboundTransferEmailVerificationCard extends React.Component {
 	refreshContactEmail = () => {
 		this.setState( { loading: true } );
 
-		checkInboundTransferStatus( this.props.selectedDomainName, ( error, result ) => {
+		getInboundTransferAdminEmail( this.props.selectedDomainName, ( error, result ) => {
 			if ( ! isEmpty( error ) ) {
 				return;
 			}
 
 			this.setState( {
 				contactEmail: result.admin_email,
+				manualWhois: result.manual_whois,
 				loading: false,
 			} );
 		} );
 	};
 
 	render() {
 		const { selectedDomainName, selectedSiteSlug, translate } = this.props;
-		const { loading, contactEmail } = this.state;
+		const { loading, manualWhois, contactEmail } = this.state;
 
 		if ( loading ) {
 			return null;
 		}
 
+		if ( manualWhois ) {
+			return (
+				<Card highlight="warning">
+					<div>
+						<h1 className="inbound-transfer-verification__heading">
+							{ translate(
+								'The authorization email is still waiting to be sent.'
+							) }
+						</h1>
+						{ translate(
+							'The contact address for this domain wasn\'t immediately available. ' +
+							'We will keep checking and send the email to initiate the transfer once we have the ' +
+							'correct address. This could take up to 24 hours. We appreciate your patience.'
+						) }
+					</div>
+				</Card>
+			);
+		}
+
+		if ( ! contactEmail ) {
```

Comment:
@danhauk I'd love an revision for this.

What I did is changed the endpoint, and added copy when we have queued the transfer, but the auth e-mail isn't out yet. :) In that case, we should show a message instead of `whois` e-mail which might be incorrect.

Reply:
@aidvu I like the additional info from the endpoint. It's a great improvement for the user. Some of the copy sounds a little awkwardly worded. Here's my attempt:

> The authorization email will be sent shortly.
> Check again in a few minutes to see which mailbox you can find the email in.

cc @ranh who may have better suggestions

Reply:
That's me trying to be cool while conveying all the info... 🤦‍♂️ 

I like your copy way more. 😆